tests/openadapt/test_gisting.py

```diff
@@ -0,0 +1,19 @@
+import modal
+import pytest
+from openadapt.ml.gist_provider.gist_provider import execute, stub
+
+
+def test_gisting():
+    """
+    A smoke test for gisting using a sequence of prime numbers
+    that also shows the compression factor for the input provided.
+    """
+    test_prompt = (
+        "Please complete the following sequence of numbers: 2, 3, 5, 7, 11, 13"
+    )
```

Comment:
Can we please modify this to test on a prompt that is larger than would normally be possible (i.e. without Gisting)?

Please save the prompt in a .j2 file rather than embedding it in the python file.